content/docs/csidriver/features/powerstore.md

```diff
@@ -730,3 +730,17 @@ The following is the list of all the attribtues supported by PowerStore CSI driv
 >Make sure that the attributes specified are supported by the version of PowerStore array used. 
 
 >Configurable Volume Attributes feature is supported with Helm.
+
+## Storage Capacity Tracking
+CSI PowerStore driver version 2.5.0 and above supports Storage Capacity Tracking.
+
+This feature helps the scheduler to make more informed choices about where to start pods which depend on unbound volumes with late binding (aka "wait for first consumer"). Pods will be scheduled on a node (satisfying the topology constraints) only if the requested capacity is available on the storage array.
+If such a node is not available, the pods stay in Pending state, i.e, not scheduled.
```

Comment:
Replace comma at after "Pending state" with a period ("."). Replace "i.e, not scheduled." with "This means they are not scheduled."

Reply:
Done.